src/Nethermind/Nethermind.Evm.Test/VirtualMachineTestsBase.cs

```diff
@@ -110,9 +110,9 @@ protected TestAllTracerWithOutput Execute(params byte[] code)
             return tracer;
         }
 
-        protected TestAllTracerWithOutput Execute(long blockNumber, long gasLimit, byte[] code, long blockGasLimit = DefaultBlockGasLimit, ulong timestamp = 0)
+        protected TestAllTracerWithOutput Execute(long blockNumber, long gasLimit, byte[] code, long blockGasLimit = DefaultBlockGasLimit, ulong timestamp = 0, byte[][] blobVersionedHashes = null)
```

Comment:
Only tests, but I would consider creating some container (class) for those parameters, to easier evolve them in the future.

Reply:
Do you mean just blobVersionedHashes? Or all the stuff like blobVersionedHashes, blobs, kzgs, proof?
blobVersionedHashes `byte[][]` type looks OK for me, may be UIn256[] could look better, but would bring redundant type conversions
blobVersionedHashes is always a block part, but blobs, kzgs and proof may be considered as a separate thing, that exists untill passed to CL. Still I'd give a try to flat transaction structure, we can easily change in the future without problems with serialization

Reply:
All parameters, when your method starts to take more than 4-5 it is time to make them into a class/struct of their own